src/main/java/org/matsim/run/LausitzScenario.java

```diff
@@ -158,6 +186,22 @@ protected void prepareScenario(Scenario scenario) {
 				link.setAllowedModes(newModes);
 			}
 		}
+
+		if (emissions == EmissionAnalysisHandling.PERFORM_EMISSIONS_ANALYSIS) {
+//			do not use VspHbefaRoadTypeMapping() as it results in almost every road to mapped to "highway"!
+			HbefaRoadTypeMapping roadTypeMapping = OsmHbefaMapping.build();
+//		the type attribute in our network has the prefix "highway" for all links but pt links.
+//		we need to delete that because OsmHbefaMapping does not handle that.
+			for (Link link : scenario.getNetwork().getLinks().values()) {
+				//pt links can be disregarded
+				if (!link.getAllowedModes().contains("pt")) {
+					NetworkUtils.setType(link, NetworkUtils.getType(link).replaceFirst("highway.", ""));
+				}
+			}
+			roadTypeMapping.addHbefaMappings(scenario.getNetwork());
```

Comment:
As also written directly to Simon: We at VSP should have our current scenarios in a state, that the HBEFA attributes are part of the input network. This is a network property and thus should not depend on the mapping in the analysis script itself.